nextcloudappstore/core/fixtures/categories.json

```diff
@@ -139,6 +139,16 @@
             "master": "dashboard"
         }
     },
+    {
+        "model": "core.categorytranslation",
+        "pk": 16,
+        "fields": {
+            "language_code": "en",
+            "name": "AI",
+            "description": "Artificial Intelligence powered apps",
```

Comment:
`Artificial Intelligence-powered apps` would be more grammatically correct, I think, but maybe it's easier to say `Artificial Intelligence apps` or `Apps powered by Artificial Intelligence` instead?